src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+html,
+body {
+  height: 100%;
+  margin: 0;
+}
+
```

Comment:
I think you meant to use .header instead of body here. If you take a look on the responsive for tablet and mobile, you'll see that it's causing a problema on the layout.

Reply:
Any suggestions on how I can do this? @raulriato

Reply:
Also, add the `box-sizing: border-box;` in the same file.

![image](https://github.com/user-attachments/assets/73fa06b3-4814-4b8a-a038-78501738a93f)